Migrate reference documentation to Antora (#3806)
reactor/reactor-core: docs/modules/ROOT/pages/aboutDoc.adoc

not always complete.

ifeval::["{backend}" == "html5"]

> TIP: To facilitate documentation edits, you can edit the current page from the `Edit this Page` link located in the upper right corner sidebar. The link opens

Reviewer: ```suggestion
TIP: To facilitate documentation edits, you can edit the current page from the `Suggest Edit` link located in the upper right corner sidebar. The link opens
```

an edit `UI` directly on `GitHub` for the main source file for the current page. These links are

> only present in the `HTML5` version of this reference guide. They look like the following link:

Reviewer: ```suggestion
only present in the `HTML5` version of this reference guide. They look like the following:
```

> link:https://github.com/reactor/reactor-core/edit/main/docs/modules/ROOT/pages/aboutDoc.adoc[Edit this Page^, role="fa fa-edit"] to xref:aboutDoc.adoc[About the Documentation].

Reviewer: ```suggestion
link:https://github.com/reactor/reactor-core/edit/main/docs/modules/ROOT/pages/aboutDoc.adoc[Suggest Edit^, role="fa fa-edit"] to xref:aboutDoc.adoc[About the Documentation].
```

> link:https://github.com/reactor/reactor-core/edit/main/docs/modules/ROOT/pages/aboutDoc.adoc[Edit this Page^, role="fa fa-edit"] to xref:aboutDoc.adoc[About the Documentation].

Reviewer: ```suggestion
link:https://github.com/reactor/reactor-core/edit/main/docs/modules/ROOT/pages/aboutDoc.adoc[Edit this Page^, role="fa fa-edit"] to make changes to xref:aboutDoc.adoc[About the Documentation] page.
```

Author: see https://github.com/reactor/reactor-core/pull/3806/commits/75f03a8aba7c45a936b4eefe26ced8fcb250275b
